### DISCUSSION TITLE
How to convert inference model to training model

### DISCUSSION THREAD

**Discussion by AlexPasqua** [PaddlePaddle/PaddleOCR#14104] [Q&A]
Hello, I see it's possible to export a model for inference through the `tools/export_model.py` script, but

is there a way to convert an inference model (`.pdiparams`) to a training one (`.pdparams`), so I can perform a fine-tuning?

Thanks in advance for clarifications 🙂

**Comment by GreatV** [comment]
no, you can't.

**Reply by AlexPasqua** [reply]
> no, you can't.

I saw online something like loading the inference model and then re-saving it, or loading its state_dict into a training model and then saving the latter... but maybe it's not possible

**Reply by AlexPasqua** [reply]
> no, you can't.

Do you think it's implementable? Maybe I can open a feature request in case
